### ISSUE TITLE
Updated analysis: modules that use output of `focal-cn-file-preparation`

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/OpenPBTA-analysis#581]
#### What analysis module should be updated and why?

With v15 of the data release, we are now including the following files from the `focal-cn-file-preparation` module:

```
consensus_seg_annotated_cn_autosomes.tsv.gz
consensus_seg_annotated_cn_x_and_y.tsv.gz
```

There are several modules that use the files that are committed to the repository (e.g., in `analyses/focal-cn-file-preparation/results`) that should get updated to use the files in `data`.

#### What changes need to be made? Please provide enough detail for another participant to make the update.

Here are the modules that make mention of these files in the [modules at a glance table](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses#modules-at-a-glance):

```
molecular-subtyping-ATRT
molecular-subtyping-chordoma
```

The module/changes that are part of #559 (`compare-gistic`) should also be updated if I had to guess.

#### What input data should be used? Which data were used in the version being updated?

```
consensus_seg_annotated_cn_autosomes.tsv.gz
consensus_seg_annotated_cn_x_and_y.tsv.gz
```
from v15

#### When do you expect the revised analysis will be completed?

~1 day – hopefully all of these code changes + rerunning each module can be addressed in a single pull request. If anything looks more complicated, it should be split up into it's own pull request. This is a lower priority than the changes required to get CI to work again, basically (#574).

#### Who will complete the updated analysis?

Likely someone from the CCDL.

**Comment by jaclyn-taroni** [comment]
I'm planning on taking care of this this week.

**Comment by jaclyn-taroni** [comment]
Closed via #717
